### ISSUE TITLE
chartValueSelected action problem

### ISSUE THREAD

**Issue by ghelfer** [danielgindi/Charts#2255]
Only positive y from x,y entry can be selected in chartValueSelected action using scatter.
DataEntries values +x,+y and -x,+y OK
DataEntries values  +x,-y and -x,-y NO

My code toast dataentry name (sample name) while tapped.

**Comment by may520** [comment]
Hello, I have encountered the same problem, is there a solution?

**Comment by liuxuan30** [comment]
Only happen for scatter chart?

**Comment by may520** [comment]
Yes,four quadrants may have scatter,and have duplicate values,scatter can not choose.
